packages/integration-karma/test/shadow-dom/event-in-shadow-tree/event-propagation.spec.js

```diff
@@ -93,8 +92,7 @@ describe('event propagation in simple shadow tree', () => {
         ]);
     });
 
-    // TODO: #1132 - Event.prototype.composedPath() doesn't return the shadow root
-    xit('propagate event from a child element in a document fragment', () => {
+    it('propagate event from a child element in a document fragment', () => {
```

Comment:
@ravijayaramappa this is failing in ie11

Reply:
@caridy this is regarding event bubbling in a disconnected dom tree.
In IE11, when an event is fired from inside of a document fragment, the event does not bubble up to the fragment. This is the native IE11 behavior.

However in IE11, @lwc/synthetic-shadow patches composed path to include the document fragment. So there is inconsistency here because the composed path says the event will bubble up to the fragment and the event stops just below the fragment.

This is the cause of this test failure. Seems like a corner case. We can write an alternative test case for IE11, to verify that listeners up to the fragment are invoked.